apps/web/vibes/soul/sections/error/index.tsx

```diff
@@ -1,14 +1,14 @@
-import { ButtonLink } from '@/vibes/soul/primitives/button-link';
+import { Button } from '@/vibes/soul/primitives/button';
 
-interface Link {
+interface Action {
   label: string;
-  href: string;
+  onRetry: () => void;
 }
 
 interface Props {
   title?: string;
   subtitle?: string;
-  cta?: Link;
+  cta?: Action;
```

Comment:
Given the use of the word "Action" for server actions and client actions I think we should avoid this name

Reply:
How about we take in a `ctaLabel` and `ctaOnClick` props?

Also, what if we always rendered the CTA button using a default label and opted for `window.location.refresh()` if a value isn't provided?